### PR TITLE
Added Portuguese string translation - value-pt

### PR DESCRIPTION
**Pull request by anselmoalexandre**
Hi @tiembo 
I hope you okay.

I was watching your video explaining how to use **Jetpack library Components** using **Sunflower Android app**, and I decided to help adding **_Portuguese language_** to this app which I found it helpful to control our plants needs.

Please, review and possibly merge this pull request into the master.
Thank you.

### CODE REVIEW

**Review by Vgbhieel — APPROVED** [review]
As a portuguese native speaker, I'd like to submit my feedback by saying that it's all fine.

File: app/src/main/res/values-pt/strings.xml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<resources xmlns:tools="http://schemas.android.com/tools">
+    <!-- Translated by Anselmo Alexandre -->
+    <string name="menu_filter_by_grow_zone">Filtrar por zona de cultivo</string>
+    <string name="my_garden_title">Meu jardim</string>
+    <string name="plant_list_title">Lista de plantas</string>
+    <string name="available" >Plantas disponíveis</string>
```
Comment (rosariopfernandes) [inline app/src/main/res/values-pt/strings.xml:23]: nit: unnecessary whitespace
```suggestion
    <string name="available">Plantas disponíveis</string>
```

File: app/src/main/res/values-pt/strings.xml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    <string name="plant_details_title">Detalhes da planta</string>
+    <string name="add_plant" >Adicionar planta</string>
```
Comment (rosariopfernandes) [inline app/src/main/res/values-pt/strings.xml:25]:
```suggestion
    <string name="add_plant">Adicionar planta</string>
```

File: app/src/main/res/values-pt/strings.xml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    <string name="added_plant_to_garden">Adicionou planta ao jardim</string>
+    <string name="garden_empty">Seu jardim esta vazio</string>
```
Comment (rosariopfernandes) [inline app/src/main/res/values-pt/strings.xml:27]: The correct spelling is with the accentuation
```suggestion
    <string name="garden_empty">Seu jardim está vazio</string>
```

File: app/src/main/res/values-pt/strings.xml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    <string name="plant_date_header">Plantada</string>
+    <string name="watered_date_header">Regado pela última vez</string>
+    <string name="menu_item_share_plant">Partilhar</string>
+    <string name="share_text_plant">Confira o %s da planta na aplicação Sunflower</string>
+    <string name="gallery_title">Fotos tirados em Unsplash</string>
```
Comment (rosariopfernandes) [inline app/src/main/res/values-pt/strings.xml:32]: I think "Imagens" sounds better in this case.
```suggestion
    <string name="gallery_title">Imagens tiradas do Unsplash</string>
```

File: app/src/main/res/values-pt/strings.xml [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+    <!-- String plurals and their related prefix / suffix strings -->
+    <string name="watering_needs_prefix">Necessidades de rega</string>
+    <plurals name="watering_needs_suffix">
+        <item quantity="one" tools:ignore="ImpliedQuantity">todos dias</item>
+        <item quantity="other">em cada %d dias</item>
```
Comment (rosariopfernandes) [inline app/src/main/res/values-pt/strings.xml:38]: I think "regar a cada 5 dias" sounds better than "regar em cada 5 dias"
```suggestion
        <item quantity="other">a cada %d dias</item>
```

File: app/src/main/res/values-pt/strings.xml [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+    </plurals>
+
+    <plurals name="watering_next">
+        <item quantity="one" tools:ignore="ImpliedQuantity">regar amanhã.</item>
+        <item quantity="other">regar dentro de %d dias.</item>
+    </plurals>
+
+    <!-- Accessibility -->
+    <string name="a11y_plant_item_image">Foto da planta</string>
+    <string name="gallery_content_description">Navegue para a tela de galeria</string>
```
Comment (rosariopfernandes) [inline app/src/main/res/values-pt/strings.xml:48]: "galeria" is a feminine noun
```suggestion
    <string name="gallery_content_description">Navegue para a tela da galeria</string>
```

**Review by rosariopfernandes — CHANGES_REQUESTED** [review]
I'm also a native Portuguese speaker.

Good work on these translations!
I just have a few comments and suggestions.

File: app/src/main/res/values-pt/strings.xml (hunk 2 above)
Comment (anselmoalexandre) [reply app/src/main/res/values-pt/strings.xml:25]: Thanks Rosario.

**Review by calren — APPROVED** [review]

**Comment by calren** [comment]
this is fantastic, thanks @anselmoalexandre , @rosariopfernandes and @Vgbhieel !

**Pull request merged**